### PR TITLE
Remove contact us form

### PR DESCRIPTION
**Pull request by dongliu0426**

### CODE REVIEW

**Comment by maggie-j-liu** [comment]
also this- https://usaco.guide/general/using-this-guide?lang=cpp#contact-us
should we just remove this section?

File: src/components/markdown/ProblemsList/ProblemListItemSolution.tsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function ProblemListItemSolution(
     contents = (
       <div className="px-4 py-2 text-sm text-gray-300 dark:text-gray-500">
         <Tooltip
-          content={`We haven't written a solution for this problem yet. If needed, request one using the "Contact Us" button!`}
+          content={`We haven't written a solution for this problem yet.`}
```
Comment (maggie-j-liu) [inline src/components/markdown/ProblemsList/ProblemListItemSolution.tsx:21]: not sure if i should replace with something like "make an issue or email us"?

Comment (dongliu0426) [reply src/components/markdown/ProblemsList/ProblemListItemSolution.tsx:21]: sure let's replace it

Comment (maggie-j-liu) [reply src/components/markdown/ProblemsList/ProblemListItemSolution.tsx:21]: 🤔 do links work in tooltips?

**Comment by dongliu0426** [comment]
> also this- https://usaco.guide/general/using-this-guide?lang=cpp#contact-us
> should we just remove this section?

idk maybe we can replace it w/ smth like
> Please help us complete this guide by giving feedback and submitting GitHub issues or emailing us at usacoguide@gmail.com. You can give feedback about anything!
If you found the guide useful, or if you got stuck while using the guide, please also let us know! :)

**Comment by dongliu0426** [comment]
#1001 :eyes:

**Comment by maggie-j-liu** [comment]
o do we want that instead of removing it?

**Comment by dongliu0426** [comment]
> o do we want that instead of removing it? and is it being worked on lol

that would be ideal

**Pull request merged**